packages/app/src/client/services/AdminCustomizeContainer.js

```diff
@@ -429,4 +442,70 @@ export default class AdminCustomizeContainer extends Container {
     }
   }
 
+
+  async deleteLogo() {
+    try {
+      const formData = {
+        _csrf:  this.appContainer.csrfToken,
+        brandLogoAttachmentId: this.state.brandLogoAttachmentId,
+      };
+      await apiPost('/attachments.removeBrandLogo', formData);
+      this.setState({
+        uploadedLogoSrc: null,
+        brandLogoAttachmentId: null,
+      });
+
+    }
+    catch (err) {
+      this.setState({ retrieveError: err });
+      logger.error(err);
+      throw new Error('Failed to delete logo');
+    }
+  }
+
+  async uploadBrandLogo(file) {
+    try {
+      const formData = new FormData();
+      formData.append('file', file);
+      formData.append('_csrf', this.appContainer.csrfToken);
+      formData.append('attachmentType', AttachmentType.BRAND_LOGO);
+      formData.append('brandLogoAttachmentId', this.state.brandLogoAttachmentId);
+      const response = await apiPost('/attachments.uploadBrandLogo', formData);
+
+      this.setState({
+        uploadedLogoSrc: response.attachment.filePathProxied,
+        brandLogoAttachmentId: response.attachment.id,
+      });
+    }
+    catch (err) {
+      this.setState({ retrieveError: err });
+      logger.error(err);
+      throw new Error('Failed to upload brand logo');
+    }
+  }
+
+  switchDefaultLogo() {
+    this.setState({ isDefaultLogo: !this.state.isDefaultLogo });
+  }
+
+  async updateCustomizeLogo() {
+    try {
+      const response = await apiv3Put('/customize-setting/customize-logo', {
+        isDefaultLogo: this.state.isDefaultLogo,
```

Comment:
Does this become `false` even if no logo is uploaded?

Reply:
`isDefaultLogo` will `true` if `uploadedLogoSrc == null` so Default Logo button will checked by default

Reply:
As shown in the attached GIF, `isDefaultLogo` becomes `false` in DB if you choose `Upload Logo` and hit `Update` button with no images uploaded. And it stays `false` until you choose `Default Logo` and hit `Update` button again.

It looks like a user has chosen default logo when no images are uploaded but, it's not the same in database.

I think you need some evaluation before saving the value of `isDaufltLogo` to database

![isdefaultlogo](https://user-images.githubusercontent.com/35527421/173321932-e4bd8c0e-ce57-45f5-ac11-5854301a3e77.gif)

Reply:
I don't think this is fixed.

This is the expectation ↓
In MongoDB -> `configs` collection -> a document with the key: `customize:isDefaultLogo` -> `value`
┗ the `value` should be `true` if a user hit `Update` button without an image uploaded.

Reply:
I have modified `updateCustomizeLogo` to set `isDefaultLogo`  to `true` if no logo uploaded

Reply:
I am sorry, but let's change the detailed design.

If user chose `Upload Logo`, then set `isDefaultLogo` to `true` when udpate. it doesn't matter if the file is uploaded or not.

After reloading the page and if there is no image uploaded(uploadedLogoSrc is null) and `isDefaultLogo` is `false`,  in mongo DB, the user should see it the screen lile this.(and that is ok because Current Logo shows the default, GROWI logo)

<img width="1254" alt="image" src="https://user-images.githubusercontent.com/35527421/174062975-62d6d97f-21eb-46a8-a8a3-aa911c42f66e.png">

If then, the user upload an image and reload the page, the user can now see the logo on  the top left, `Upload Logo` is chosen,  and the Current Logo shows the uploaded logo with `Delete Logo` button with it.

Reply:
I have done a test based on this doc https://docs.google.com/spreadsheets/d/1NSvoD5NNOfMZWzcxZKYjKoG6AIkw8wZ5IEWlOS3Nzuk/edit?usp=sharing
## Testing with pattern 1 (Select `Default logo` without image uploaded)
### Result
![p1-upload](https://user-images.githubusercontent.com/92426728/174240023-07cdc9dd-ab3c-4cde-8d15-d3a277815f79.png)
![p1-logo](https://user-images.githubusercontent.com/92426728/174240035-a06c4798-77f4-4c3d-8ffa-0ab818e6f15d.png)
![p1-db](https://user-images.githubusercontent.com/92426728/174240040-8c0b3bfa-3a19-4655-822c-4268a81a1991.png)

## Testing with pattern 2 (Select `Default Logo` with image uploaded)
### Result
![p2-upload](https://user-images.githubusercontent.com/92426728/174240192-b71dbea8-d0a8-4b82-b71b-91a63a330bb3.png)
![p2-logo](https://user-images.githubusercontent.com/92426728/174240198-6923377e-4fb8-445e-9a38-a26c5f5f5726.png)
![p2-db](https://user-images.githubusercontent.com/92426728/174240195-c72738b7-1961-4755-b6dc-043ca1da026b.png)

## Testing with pattern 3 (Select `Upload logo` without image uploaded)
### Result
![p3-upload](https://user-images.githubusercontent.com/92426728/174240495-cf5f8e3b-374d-43c0-a029-6f86e275a486.png)
![p3-logo](https://user-images.githubusercontent.com/92426728/174240491-864fa807-50f3-4957-8919-a99a96d6fac9.png)
![p3-db](https://user-images.githubusercontent.com/92426728/174240487-2ccdb877-22c2-46f6-b7a7-42eeb0f4f2b8.png)

## Testing with pattern 4 (Select `Upload Logo`  with image uploaded)
### Result 
![p4-upload](https://user-images.githubusercontent.com/92426728/174240783-6995541c-fbc7-4c23-a27c-c915708b22d8.png)
![p4-logo](https://user-images.githubusercontent.com/92426728/174240781-cff16d73-cd36-4dd1-a120-a7f47e0988d3.png)
![p4-db](https://user-images.githubusercontent.com/92426728/174240776-49b50ae5-fdb7-4800-ac11-fdddbd43eb59.png)